grammar.js

```diff
@@ -177,29 +177,31 @@ module.exports = grammar({
     // Intended for table column names per `:help help-writing`.
     // TODO: children should be $.word (plaintext), not $.atom.
     column_heading: ($) => seq(
-      field('name', seq(choice($._atom_noli, $._uppercase_words), repeat($._atom))),
+      alias(seq(choice($._atom_noli, $._uppercase_words), repeat($._atom)), $.heading),
       '~',
       token.immediate('\n'),
     ),
 
     h1: ($) =>
-      seq(
-        token.immediate(field('delimiter', /============+[\t ]*\n/)),
-        repeat1($._atom),
+      prec(1, seq(
```

Comment:
I believe TS considers declaration order as part of precedence. So possibly we could avoid `prec(1,...)` by declaring these headings before `block`. But that could be tried later.

Reply:
I could make it tighter, but _some_ precedence is needed to resolve the conflict between heading and possible taglinks.

Reply:
> I believe TS considers declaration order as part of precedence.

Only for terminals, e.g. string literals and regex patterns, and strings are higher than regex patterns by default